index.js

```diff
@@ -1,45 +1,14 @@
-const express = require("express");
-const config = require("./config.js");
-const axios = require("axios");
-const Discord = require("discord.js");
-
-const admin = require("firebase-admin");
-let serviceAccount;
-if (process.env.FIREBASE_SERVICE_ACCOUNT) {
-  serviceAccount = JSON.parse(process.env.FIREBASE_SERVICE_ACCOUNT);
-} else {
-  serviceAccount = require("./yu-gi-oh-inventory-firebase-adminsdk-z8h62-7925253eaa.json");
-}
-
-admin.initializeApp({
-  credential: admin.credential.cert(serviceAccount),
-  databaseURL: "https://yu-gi-oh-inventory.firebaseio.com",
-});
-
-const db = admin.database();
+import { BOT_TOKEN } from "./config.js";
+import express from "express";
+import Discord from "discord.js";
+import drawCommand from "./drawCommand.js";
+import inventoryCommand from "./inventoryCommand.js";
+import lookUpCommand from "./lookupCommand.js";
+import removeCardCommand from "./removeCardCommand.js";
+import helpCommand from "./helpCommand.js";
```

Comment:
since these files have a natural grouping (they're all commands), consider putting them into a `commands/` folder for better organization